src/main/kotlin/gg/skytils/skytilsmod/features/impl/dungeons/ScoreCalculation.kt

```diff
@@ -67,17 +67,17 @@ object ScoreCalculation {
         "F1" to FloorRequirement(.3),
         "F2" to FloorRequirement(.4),
         "F3" to FloorRequirement(.5),
-        "F4" to FloorRequirement(.6),
-        "F5" to FloorRequirement(.7, 10 * 60),
+        "F4" to FloorRequirement(.6, 12 * 60),
```

Comment:
I believe last time they lied about what they changed or something, are you sure they changed it

Reply:
I'm not so sure myself, sadly I cannot test master ones, but I will run a few F1~F6s to make sure those are correct

Reply:
I've done some F4 and F6 runs, and speed score doesn't go down until 12 min+

Reply:
As for master floors, I'm justr gonna keep the original values since I can't test them, but Entrance speed score calculation is kinda different: from my test runs, it seems like no speed score reduction is applied until 20min+(varying from 21min~23min), so I'm setting speed score requirement to 20min